src/renderer/scss/component/_modal.scss

```diff
@@ -31,6 +30,11 @@
   box-shadow: var(--box-shadow-layer);
   max-width: var(--modal-width);
   word-break: break-word;
+
+  .btn.btn--alt {
+    // Set modal buttons bg color
+    background-color: var(--modal-btn-bg-color);
+  }
```

Comment:
@seanyesmunt Not sure if this is the right way to do this, but I don't have time to add a new class to all the `btn-alt` used inside the modal component :stuck_out_tongue:

Reply:
I think this is ok for now.

Reply:
Well, do modals in dark mode not use the regular alt button style?

Reply:
@seanyesmunt yes but it's the same color as the background of the modal :upside_down_face:,
see https://design.lbry.io